### PR TITLE
Vra 118 soundchannel error msg

### PR DESCRIPTION
**Pull request by nyy23**

### CODE REVIEW

File: VocalRaidAssistant.lua [hunk 1]
```diff
@@ -165,10 +165,19 @@ end
...
 	local soundFile = "Interface\\AddOns\\VocalRaidAssistant\\Sounds\\" .. profile.sound.soundpack .. "\\" .. spellID ..
 									".ogg"
+
+	local channel = profile.sound.channel
 	if soundFile then
-		local success = PlaySoundFile(soundFile, addon.SOUND_CHANNEL[profile.sound.channel])
-		if not success and GetCVar("Sound_EnableAllSound") ~= "0" then
-			print(format("VRA - Missing soundfile for configured spell: %s, Voice Pack: %s", GetSpellInfo(spellID),profile.sound.soundpack))
+		local success = PlaySoundFile(soundFile, channel)
+		if not success then
```
Comment (Nitrak) [inline VocalRaidAssistant.lua:172]: Won't this spam the user until they turn in sound again?

There could be a reason to temporarily turn off sound. If you can add a buffer-time between each message (say 10 minutes) so we don't spam the user.

The missing for configured spell can be set without any buffer-time if you feel that's more appropriate. The issue with this one if it has a buffer-time is that we would need to track "broken" abilities and buffer-time on them.

**Review by Nitrak — CHANGES_REQUESTED** [review]

File: VocalRaidAssistant.lua (hunk 1 above)
Comment (nyy23) [reply VocalRaidAssistant.lua:172]: Valid point, this will produce a lot messages.
We can work with some 'throttling' here.

Comment (Nitrak) [reply VocalRaidAssistant.lua:172]: I set up a changelog note for 2.3.2 for when this PR is complete.

Comment (nyy23) [reply VocalRaidAssistant.lua:174]: Added type specific throttling.
For type 'msg' 15s is set, 'sound' is user configured

**Comment by Nitrak** [comment]
@nyy23 you think 15 seconds is enough time? Again, if someone turns off the sound for some reason a msg every 15 seconds seems like quite a lot.

Let's try to put it at 60 seconds initially.

**Comment by nyy23** [comment]
Yes, is think 15s in a good value. 
Keep in mind, the amount of spam Addons or Weakauras produce. Propagating one error message every 60s might get "pushed" out of chat very quickly.

**Comment by Nitrak** [comment]
That is with that kept in mind. This error message is just to remind the user that if he/she wants the add-on to function it needs to be turned on. Once every 15 seconds is quite often for a message like this.

**Comment by Nitrak** [comment]
I am working on merging Develop into this atm. I found some errors on Develop, so please send me a PM before you do anything on this.

**Pull request merged**